### DISCUSSION TITLE
create function triggered twice when "Add...option" is clicked on

### DISCUSSION THREAD

**Discussion by jcraw62** [selectize/selectize.js#2037] [Q&A]
With v 0.15.2 the create function is triggered twice if instead of hitting enter you mouse click on the "Add...option" label.

Is this a bug? Is there a work-around?

Maybe the real question is is "Add...option" selected so that enter should work (and click merely re-selects the new option with no effect)

**Comment by JackJoeJack** [comment]
I found a solution for that version (0.15.2)

Go to this line of code:
`$dropdown.on('mouseup click', '[data-selectable]', function() { return self.onOptionSelect.apply(self, arguments); });`

Remove the "mouseup":
`$dropdown.on('click', '[data-selectable]', function() { return self.onOptionSelect.apply(self, arguments); });`

Works!

**Reply by sebastian-palma** [reply]
This works <3. Just to be explicit, this change must be performed in whatever file holds the code for selectize. If a minified version of the code, you'll see something like this

```js
r.on("mouseup click","[data-selectable]",function(){return e.onOptionSelect.apply(e,arguments)})
```
